modules/oe_theme_helper/src/Plugin/Field/FieldFormatter/FeaturedMediaFormatter.php

```diff
@@ -152,6 +152,8 @@ public function viewElements(FieldItemListInterface $items, $langcode) {
    *
```

Comment:
We had a ticket "Contact body should take full width when no image is present" https://citnet.tech.ec.europa.eu/CITnet/jira/browse/EWPP-716. 

Currently wrapper is shown around absent image in the contact entity when media image is unpublished. Please check condition in templates/oe_contact/oe-contact--full.html.twig.

<img width="1658" alt="Screenshot 2021-05-31 at 12 11 33" src="https://user-images.githubusercontent.com/1183951/120170949-2881f880-c20a-11eb-932a-044118fbf31a.png">